src/seedu/addressbook/data/person/Unit.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Unit {
+
+    private final String UNIT;
+
+    /**
+     * Validates given address.
+     *
+     * @throws IllegalValueException if given address string is invalid.
+     */
+    public Unit(String unit){
+    	UNIT = unit;
+    }
+    
+    /**
```

Comment:
The start of the header comment does not follow the convention (i.e. first sentence should be something like `Retrieves  the BLOCK …` instead of `Retrive the BLOCK or `This method is for retrieving…`)